.gitignore

```diff
@@ -1,3 +1,7 @@
+# Python Virtual Environments
+*.virtenv
+*.venv
+
```

Comment:
Per our discussion I'm going to leave this, but I'm not strongly tied to the idea either.